app/src/main/java/com/sun_asterisk/foodies/data/source/remote/fetchjson/ParseDataWithJson.kt

```diff
@@ -47,12 +49,27 @@ class ParseDataWithJson {
         return data
     }
 
+    fun parseJsonToData(jsonObjectArray: JSONArray?, keyEntity: String): Any {
```

Comment:
```suggestion
    fun parseJsonToData(keyEntity: String, jsonObjectArray: JSONArray?): Any {
```
Nên để key lên trước cho dễ hiểu là mình đang parse Json của thằng nào

Reply:
Dạ